cypress/e2e/models/migration/issues/issues.ts

```diff
@@ -70,4 +73,16 @@ export class Issues {
             click(selector);
         }
     }
+
+    public static validateFilter(issues: AppIssue[], filterType: filterIssue, item: string): void {
```

Comment:
Optional : 
item can be renamed to filterValue .

Reply:
Done